overlord/devicestate/handlers_bootconfig.go

```diff
@@ -78,15 +78,19 @@ func (m *DeviceManager) doUpdateManagedBootConfig(t *state.Task, _ *tomb.Tomb) e
 	if err != nil {
 		return fmt.Errorf("cannot update boot config assets: %v", err)
 	}
+
+	// minimize wasteful redos
+	t.SetStatus(state.DoneStatus)
```

Comment:
I see, we would need to keep this one, otoh the function below setting it again is not a problem either